client/src/GlobalStyles.js

```diff
@@ -60,6 +60,18 @@ h1,
     margin: 0;
   }
 
+
+
+  ul,
+  li {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  
+ 
+
+
```

Comment:
Why so many blank lines? I think your Prettier needs configuration (<kbd>shift</kbd> + <kbd>command</kbd> + <kbd>p</kbd> --> Format Document AND check "Format on save" in the Settings).